RunningOrder/ToolbarManager.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  ToolbarManager.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 27/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import Foundation
+import Cocoa
+import SwiftUI
+
+class ToolbarManager: NSObject, ObservableObject, NSToolbarDelegate, NSToolbarItemValidation {
+
+    let windowContainer: AppWindowController
```

Comment:
could you abstract the logic of this : we don't need aan AppWindowController, just a class that store a splitVC

Reply:
with a protocol

Reply:
Or we can just store the split viewController,
```
 init(windowContainer: AppWindowController) {
        splitViewController = (windowContainer.window?.contentView?.subviews.first?.subviews.first?.subviews.first as? NSSplitView)?.delegate as? NSSplitViewController
    }
```

Reply:
Where split viewController is a let properties